src/sap/fhir/model/r4/FHIRUtils.js

```diff
@@ -580,9 +580,8 @@ sap.ui.define([
 
 		if (sFilterType == FilterType.Application) {
 			oBinding.aApplicationFilters = aFilters;
-		} else {
-			oBinding.aFilters = aFilters;
```

Comment:
why removing the aFilters here and adding them in the other condition?

Reply:
Who evalutes the oCombinedFilter?

Reply:
@flovogt : combined filter is being evaluated by default in list binding https://github.com/SAP/openui5/blob/master/src/sap.ui.core/src/sap/ui/model/ListBinding.js#L428. This pr is fix wherein a regression caused while making changes for this https://github.com/SAP/openui5-fhir/pull/265. The aFilters were by default present in the other condition